Sources/OktaOidc/Common/OktaOidcConfig.swift

```diff
@@ -30,7 +30,9 @@ public class OktaOidcConfig: NSObject {
      requests throughout OktaOidc. More information could be found here: https://github.com/okta/okta-oidc-ios/blob/master/README.md#modify-network-requests.
      */
     @objc public weak var requestCustomizationDelegate: OktaNetworkRequestCustomizationDelegate?
-
+    
```

Comment:
Can we name a variable like `tokenValidator` or `customTokenValidator`? I believe `okta` may be omitted.